### DISCUSSION TITLE
LDAP: Logged in User suddenly changed to another user in domain

### DISCUSSION THREAD

**Discussion by mesuttalebiacerpro** [requarks/wiki#7103] [Error / Bug Report]
While I was working on a page, and editing it's content, suddenly errors appears. so I refreshed the page and the logged in user changed to another user in our domain.

![image](https://github.com/requarks/wiki/assets/59725815/a5658c67-c815-46f2-9f12-027317bfde9e)

I look at the management and see, that the user logged in in the same time that I got errors.

![image](https://github.com/requarks/wiki/assets/59725815/c3763bb8-b24a-45ab-b496-323529b3ccac)

**Comment by mus65** [comment]
This is probably caused by server-side caching, see https://github.com/requarks/wiki/discussions/6822#discussioncomment-9533304
